.github/workflows/test-getting-started.yml

```diff
@@ -0,0 +1,39 @@
+name: Test the getting start steps
+
+# run CI on pushes to main, and on all PRs (even the ones that target other
+# branches)
+
+on:
+ push:
+   branches: [$default-branch]
```

Comment:
this doesnt seem to be valid syntax https://github.com/orgs/community/discussions/26597#discussioncomment-3252475

Reply:
```suggestion
   branches: [main]
```

Reply:
Nice catch. Fixed it.